app/Http/Controllers/Assets/AssetsController.php

```diff
@@ -338,27 +344,39 @@ public function update(AssetRequest $request, $assetId = null)
         $asset->notes        = $request->input('notes');
         $asset->physical     = '1';
 
+        \Log::debug('The sky is blue.');
```

Comment:
Yeah, that's still in there because that section isn't done. That's the weird file API thing for assets that's still a leftover thing to be resolved. :(